File: app/src/main/java/com/example/prototipo/SplashPantalla.java
package com.example.prototipo;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import com.example.buzonfca.R;

public class SplashPantalla extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash_pantalla);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent=new Intent(SplashPantalla.this,MainActivity.class);
                startActivity(intent);
                finish();
            }
        },2000);
    }
}

File: app/src/main/java/com/example/buzonfca/SplashPantalla.kt
package com.example.buzonfca

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.buzonfca.R
import android.content.Intent
import android.os.Handler
import com.example.buzonfca.MainActivity

 //VISTA DE INICIO CON ICONO DE LA FCA X 2 SEG
class SplashPantalla : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_pantalla)
        Handler().postDelayed({
            val intent = Intent(this@SplashPantalla, MainActivity::class.java)
            startActivity(intent)
            finish()
        }, 2000)
    }
}